engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActorSpec.scala

```diff
@@ -88,7 +91,11 @@ class WorkflowExecutionActorSpec extends CromwellTestkitSpec with BeforeAndAfter
 
       val workflowId = WorkflowId.randomId()
       val engineWorkflowDescriptor = createMaterializedEngineWorkflowDescriptor(workflowId, SampleWdl.SimpleScatterWdl.asWorkflowSources(runtime = runtimeSection))
-      val workflowExecutionActor = system.actorOf(WorkflowExecutionActor.props(workflowId, engineWorkflowDescriptor, serviceRegistry, jobStore, dockerHashLookupActor, AllBackendInitializationData.empty, restarting = false), "WorkflowExecutionActor")
+      val workflowExecutionActor = system.actorOf(
+        WorkflowExecutionActor.props(
+          workflowId, engineWorkflowDescriptor, serviceRegistry, jobStore,
+          AllBackendInitializationData.empty, restarting = false),
```

Comment:
ToL: I like to label the actors going into this constructor (e.g. `serviceRegistryActor = serviceRegistry, jobStoreActor = jobStore`) - and in fact there was an instance in my PR where a probe was going in as the wrong parameter because they're all actorRefs and horribly easy to mis-order